config/local-dev.yml

```diff
@@ -7,7 +7,13 @@ analysis:
     metadata_registry:
       interval: 1h
     aggregate_stats:
-      interval: 10m
+      tx_volume_interval: 10m
+      daily_active_accounts_window_step: 5m
```

Comment:
I thought about this, and I didn't find it that weird. The indexer would start at `last_analyzed_slot + floor(current_step)` unless I missed something.   

So for example, changing from `5m` to `10m` there would be a point in DB where the time between recorded stats would start being `10m`, previously `5m`.  Since all the API response buckets are timestamped, and the bucket size remains the same, I imagine the consumers would likely be minimally affected by such a change.


However, I'm also not against just hardcoding this.

Reply:
Actually, by adding support for `window_step_seconds` to the API and this also effecting what can be queried, maybe moving this into a constant does indeed make more sense.